json_schema_for_humans/templates/content.html

```diff
@@ -51,6 +52,10 @@
                 {{ " " }}<span class="badge badge-success default-value">Default: {{ default_value }}</span>
             {%- endif -%}
 
+            {%- if kw_additional_properties and kw_additional_properties.literal == false -%}
```

Comment:
I'm unfamiliar with the this template stuff, and I was surprised that I needed to compare the `.literal` with `false` -  I was expecting it to be the pythonic `False`, similar to how we set variables to `False` further down. Why is this the case? It feels like I'm writing python but not quite.

Same question for line 142 where I had to check for `true`.

Reply:
I was mostly confused since in the SchemaNode class, `literal` is typed as `Union[str, int]`, so I was trying to compare with `"false"` initially.

Reply:
This seems to be some [jinja2 magic](https://jinja.palletsprojects.com/en/2.9.x/templates/#literals)

> The special constants true, false, and none are indeed lowercase. Because that caused confusion in the past, (True used to expand to an undefined variable that was considered false), all three can now also be written in title case (True, False, and None). However, for consistency, (all Jinja identifiers are lowercase) you should use the lowercase versions.

Contrary to what it says, I do prefer uppercase for the same reason as you, I want to write Python :P

> I was mostly confused since in the SchemaNode class, literal is typed as Union[str, int], so I was trying to compare with "false" initially.

Sorry about that, you can correct it to `Union[str, int, bool]` (or let me do it, I don't mind).

Reply:
I made the changes

Reply:
Cool thanks.

> > ...all three can now also be written in title case

I could have sworn I did try with just upper case literals and it didn't work, but maybe I had something else broken at the same time. Oh well, it clearly does work, so never mind!

I'll push an example for the docs tomorrow (about to finish up for the day), then we should be good to merge.